Makefile.toml

```diff
@@ -151,6 +152,26 @@ workspace = false
 install_script = ["(cd ./api_tests; yarn install;)"]
 script = ["cd api_tests; yarn run check"]
 
+[tasks.cargo-toml-format]
+description = "Runs cargo tomlfmt to format Cargo.toml files."
+workspace = false
+install_crate = { crate_name = "cargo-tomlfmt", binary = "cargo-tomlfmt", test_arg = "--help" }
+script = [
+'''
+find . -name Cargo.toml | xargs -n1 cargo tomlfmt -p
+'''
+]
+
+[tasks.check-cargo-toml-format]
+description = "Runs cargo tomlfmt to check appropriate Cargo.toml format."
+workspace = false
+install_crate = { crate_name = "cargo-tomlfmt", binary = "cargo-tomlfmt", test_arg = "--help" }
+script = [
+'''
+find . -name Cargo.toml | xargs -n1 cargo tomlfmt -d -p
+'''
+]
+
```

Comment:
Duplicate tasks?

Reply:
Hum no, as per all other format tasks. when checks, the other one formats.
See `--help`:
```
▶ cargo tomlfmt --help
cargo-tomlfmt-tomlfmt 
It's just an alias as a subcommand of cargo.

USAGE:
    cargo-tomlfmt tomlfmt [FLAGS] [OPTIONS]

FLAGS:
    -c, --create     Create a formatted manifest as Cargo.toml.new when dryrun.
    -d, --dryrun     Do NOT overwrite the file.
    -h, --help       Prints help information
    -k, --keep       Keep the original manifest as Cargo.toml.bak.
    -V, --version    Prints version information

OPTIONS:
    -p, --path <path>    Path to the manifest. (default is Cargo.toml)
```

`-d` is for a dry run (only checks)